migrator/migrations/m_186_to_m_187_add_blob_search/migration.go

```diff
@@ -0,0 +1,85 @@
+package m186tom187
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	afterSchema "github.com/stackrox/rox/migrator/migrations/m_186_to_m_187_add_blob_search/schema/after"
+	beforeSchema "github.com/stackrox/rox/migrator/migrations/m_186_to_m_187_add_blob_search/schema/before"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/utils"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 186,
+		VersionAfter:   &storage.Version{SeqNum: 187},
+		Run: func(databases *types.Databases) error {
+			err := convert(databases.GormDB)
+			if err != nil {
+				return errors.Wrap(err, "moving persistent files to blobs")
+			}
+			return nil
+		},
+	}
+	batchSize = 2000
+	log       = logging.LoggerForModule()
+)
+
+func convert(db *gorm.DB) (err error) {
```

Comment:
In looking at this, I think this schema update and migration ARE COMPATIBLE with the previous release.  Is that assessment correct?

Reply:
Yes.